docs/sql-migration-guide-upgrade.md

```diff
@@ -9,6 +9,8 @@ displayTitle: Spark SQL Upgrading Guide
 
 ## Upgrading From Spark SQL 2.4 to 3.0
 
+  - Since Spark 3.0, to parse decimals in locale specific format from CSV, set the `locale` option to proper value.
```

Comment:
While we are here, let's format the migration guide item format as others. like ..

In Spark version 2.4 and earlier, it only deals with locale specific decimal notation like `,`. Since Spark 3.0, the locale can be set by `locale` and default locale is ....

Please feel free to change as you think is righter